apm-dist-es7/src/main/assembly/binary-es7.xml

```diff
@@ -59,6 +65,8 @@
                 <include>zabbix-rules/*.yaml</include>
                 <include>otel-oc-rules/*</include>
                 <include>ui-initialized-templates/*</include>
+                <include>lal/*</include>
+                <include>log-mal-rules/*</include>
```

Comment:
@wu-sheng these two directories are gone (in git and dist tar) because they are empty, I need to create something like `.gitignore` file in these directories to make `git add` check the directories in, WDYT?

Reply:
I think you could keep 2 empty LAL and MAL files in it?